core/src/saros/account/XMPPAccountStore.java

```diff
@@ -318,35 +293,55 @@ public int compare(XMPPAccount a, XMPPAccount b) {
    * the server list contains
    *
    * <ul>
-   *   <li>jabber.org
    *   <li>googlemail.com
-   *   <li>saros-con.imp.fu-berlin.de
    * </ul>
    *
    * @return
    */
   public synchronized List<String> getServers() {
-    List<String> servers = new ArrayList<String>();
-    for (XMPPAccount account : accounts) {
-      String server = account.getServer();
-      if (!servers.contains(server)) servers.add(server);
+
+    final Set<String> servers = new HashSet<String>();
+
+    for (final XMPPAccount account : getAllAccounts()) servers.add(account.getServer());
+
+    return new ArrayList<String>(servers);
+  }
+
+  /**
+   * Sets the given account as the default one.
+   *
+   * @param account the account to set as default or <code>null</code>
+   * @throws IllegalArgumentException if the account is not found in the store
+   */
+  public void setDefaultAccount(final XMPPAccount account) {
+    synchronized (this) {
+      if (account != null && !accounts.contains(account))
+        throw new IllegalArgumentException(
+            "account '" + account + "' is not in the current account store");
+
+      defaultAccount = account;
+
+      saveAccounts();
     }
-    return servers;
+
+    notifyActiveAccountListeners();
   }
 
   /**
    * Makes the given account active.
    *
    * @param account the account to activate
    * @throws IllegalArgumentException if the account is not found in the store
+   * @deprecated Will be removed soon, do not use
```

Comment:
Refer to `setDefaultAccount(XMPPAccount)` as an alternative? (hinges on my general question on the meaning of "default" vs "active")

Also, this seems to do exactly the same as `setDefaultAccount(XMPPAccount)` minus allowing `null` values. Why not just replace the old calls with the new method directly and remove this method (again, in the separate refactoring commit)?

Reply:
Because somehow Gradle fucked up my Eclipse Build Path for IntelliJ, I am currently unable to compile it because of every library entry exists twice.

To be more specific: soon = not the normal student soon aka never but right after the commit that follows this commit.

Reply:
> Because somehow Gradle fucked up my Eclipse Build Path for IntelliJ, I am currently unable to compile it because of every library entry exists twice.

Do you mean your build path for the IntelliJ project or your build path/config in IntelliJ (i.e. for building Saros in IntelliJ IDEA)?

Reply:
![image](https://user-images.githubusercontent.com/41400679/62393877-cb647f80-b56b-11e9-8d27-56b366957172.png)

Reply:
Hmm. Delete the directory containing the gradle cache and build again? Also, did this affect other parts of the config (tracked by git)?

Reply:
I delete the .project file and called gradle again, same issue. I manually edited the .project and now thinks works. TBH I do not get those mass amounts of libraries. IIRC you only need about 20 libraries to compile IntelliJ in the Eclipse IDE.